opencog/utils/valueUtils.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _OPENCOG_VALUE_UTILS_H_
+#define _OPENCOG_VALUE_UTILS_H_
+
+#include <opencog/atoms/value/Value.h>
+#include <opencog/atoms/base/Link.h>
+#include <opencog/combo/combo/vertex.h>
+
+namespace opencog
+{
+using namespace combo;
+
+inline bool bool_value_to_bool (const ValuePtr &valuePtr)
+{
+	return valuePtr->get_type() == TRUE_LINK;
+}
+
+inline ValuePtr bool_vertex_to_value (const vertex &v)
+{
+	return ValuePtr(vertex_to_bool(v) ? createLink(TRUE_LINK) : createLink(FALSE_LINK));
```

Comment:
I'm thinking you probably don't need `createLink` either, but maybe it's simpler for now.

Reply:
Wait, obviously these links should be static.

Reply:
@kasimebrahim, I think it's worthwhile to make `ValuePtr(createLink(TRUE_LINK))` and `ValuePtr(createLink(FALSE_LINK))` static. Maybe it's an unnecessary optimization, but I feel otherwise. What do you think?

Reply:
Yeah that is a good idea.